lib/screens/profile_screens/edit_name/interactor/viewmodels/edit_name_event.dart

```diff
@@ -1,37 +1,8 @@
 part of 'edit_name_bloc.dart';
 
-abstract class EditNameEvent extends Equatable {
-  const EditNameEvent();
-
-  @override
-  List<Object?> get props => [];
-}
-
-class OnNameChanged extends EditNameEvent {
-  final String name;
-
-  const OnNameChanged({required this.name});
-
-  @override
-  List<Object?> get props => [name];
-
-  @override
-  String toString() => 'OnNameChanged { name: $name }';
-}
-
-class SubmitName extends EditNameEvent {
-  const SubmitName();
-
-  @override
-  List<Object?> get props => [];
-
-  @override
-  String toString() => 'SubmitName';
-}
-
-class ClearPageCommand extends EditNameEvent {
-  const ClearPageCommand();
-
-  @override
-  String toString() => 'ClearPageCommand';
-}
+@freezed
```

Comment:
how cool is this? 

We went from 37 lines of code to just 8

Reply:
Looks great I just wonder If we are still getting the event logs in console with this improvement.